docs/guides/sdk/getting-started.md

```diff
@@ -252,6 +252,72 @@ All SDK composables can also be conveniently imported from the root package `@di
 
 :::
 
+## Global APIs
+
+To keep the SDK dependency-free, it does rely on the APIs mentioned below, which originally came from the browser
+ecosystem and may not be available in all environments.
+
+#### The `fetch` API
+
+There are various reasons why you would want to overwrite or set `fetch` if it's not available. For example, Node.js
+version 16 and below do not yet ship `fetch`.
```

Comment:
Node.js >= 18 is required for the SDK anyway, therefore adjusting this sentence:
```suggestion
This API is shipped with almost every modern runtime. Nevertheless, there might be reasons to overwrite or set the
implementation, for example, if an alternative implementation is preferred or if you actually work with a special
runtime where `fetch` is not available.
```